test/main.go

```diff
@@ -3,7 +3,7 @@ package main
 import (
 	"fmt"
 
-	"github.com/thedatashed/xlsxreader"
+	"github.com/nathj07/xlsxreader"
```

Comment:
same here - This should still be `thedatashed`

Reply:
Apologies, it should be that indeed. Let me fix this up